wolf-data/src/Wolf/Data/Prompt.hs

```diff
@@ -1,4 +1,4 @@
-module Wolf.Cli.Prompt
+module Wolf.Data.Prompt
```

Comment:
What?! Why?!

No No No!

Promptiing makes no sense for other reverse dependencies. Leave this code where it belongs.

Reply:
The reason I did this, is to avoid wolf-data to depend on wolf-cli. The options are:
1. Put the code in wolf data
2. Make wolf data dependent on wolf-cli (which means there's a cyclic dependency, probably causing problems in the long run, as you saw with validity and zifter)
3. Duplicate the code, and maintain it in two places
4. Put the code in a separate repository, and put that on hackage and stackage.

In my opinion, option 2 is the best one. What do you think?

Reply:
Why do you need prompt in data? If you do, something is wrong.